ch04_Het_Beheer_en_Ontwikkelmodel.md

```diff
@@ -61,7 +61,7 @@ gaat hier verder op in.
 
 Onder de genoemde activiteiten verstaan we het volgende:
 
-**Strategie**: Richtinggevende activiteiten gerelateerd aan de strategische (lange) termijn:
+**Strategie**, Richtinggevende activiteiten gerelateerd aan de strategische (lange) termijn:
```

Comment:
Die dubbele punt hoort daar gewoon. Het is niet Strategie of een richtingevende activiteit.

Reply:
![image](https://github.com/user-attachments/assets/f67e766e-1763-4561-a7d4-9e210e505e85)

waarom is er voor gekozen om Tactiek en Operationeel met komma te schrijven en neit dubbelepunt. Ik streef nog steeds tot 1 format met deze change.

Overigens als we kiezen voor een ":"  dan staan er 2 van deze tekens in 1 regel wat het lezen niet makkelijker maakt naar mijn mening maar dat kan ik fout hebben.